tests/unit/LedgerConsentContinuationOut.spec.ts

```diff
@@ -32,7 +37,7 @@ describe('Display ConsentContinuationOut component', () => {
       .toContain('FILED AND PAID  (filed by  on [unknown])  EFFECTIVE as of [unknown]')
     expect(wrapper.find('.expand-btn').text()).toContain('View Documents')
     expect(wrapper.find('.v-expansion-panel-content').exists()).toBe(false)
-    expect(vm.expiry).toEqual('[unknown]')
+    expect(vm.expiry).toEqual('August 17, 2023 at 12:59 am Pacific time')
```

Comment:
The code is working correctly but the example is wrong. Expiry at 12:59 AM makes no sense. It should expire at the end of the day -- 11:59 PM.

Ditto the test below.

Reply:
Sorry that is totally my bad

Reply:
Both tests look great.